src/core/webidl-index.js

```diff
@@ -29,7 +29,20 @@ export function run() {
     }
     idlIndexSec.insertAdjacentElement("afterbegin", header);
   }
-  if (!document.querySelector("pre.idl")) {
+
+  //filter out the IDL marked with class="exclude" and the IDL in non-normative sections
+  function idlFilter(idl) {
+    if (idl.closest(".informative")) {
```

Comment:
We can simply return it like this (without explicitly saying return `true` or `false`):
``` js
return !idl.closest(".informative");
```
Maybe we should inline it below in filter itself.

Reply:
Agree, we can probably get rid of this function entirely and just inline: 

```JS
.filter(idl => !idl.closest(".informative"));
```